### PR TITLE
[MP-19] - Create Card component

### PR DESCRIPTION
**Pull request by DiegoVillasenorVilla**
PR checklist

### Description

- Create reusable card component with the next props:
  - width, height, children and classname(optional)
- Create small and children history

image
image

### Testing Checklist

- [x] Test cases have been added/updated
- [x] Tested locally in browser (at least one other than Chrome)
- [x] All tests pass

### What JIRA Ticket(s) does this PR address?

- [MP-19](https://cd95.atlassian.net/browse/MP-19)

### CODE REVIEW

File: src/components/card/Card.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,143 @@
+import { ComponentMeta, ComponentStory } from '@storybook/react';
+import { Card } from './Card';
+import { Button } from '../Button/Button';
+import image1 from '../../assets/image1.svg';
```
Comment (caarlosdamian) [inline src/components/card/Card.stories.tsx:4]: you can add and index in /assets/ 

export all of the assests from there

File: src/components/card/Card.tsx [hunk 2]
```diff
@@ -0,0 +1,28 @@
+import React, { ReactNode } from 'react';
+import './card.css';
+
+interface CardProps {
+  children?: ReactNode;
```
Comment (caarlosdamian) [inline src/components/card/Card.tsx:5]: this is not optional

Comment (caarlosdamian) [reply src/components/card/Card.tsx:5]: JSX element

File: src/components/card/card.css [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+  align-items: center;
+  justify-content: center;
+  border: 1px solid #272727;
+  border-radius: 8px;
+  font-family: 'Nunito Sans', 'Helvetica Neue', Helvetica, Arial, sans-serif;
+  font-size: 12px;
+}
+
+.name-paragraph {
+  font-family: 'Nunito Sans', 'Helvetica Neue', Helvetica, Arial, sans-serif;
```
Comment (caarlosdamian) [inline src/components/card/card.css:47]: this can be a variable on the top

File: src/components/card/Card.stories.tsx [hunk 4]
```diff
@@ -0,0 +1,143 @@
...
+export default {
+  title: 'Components/Card',
+  component: Card,
+  args: {
+    widthSize: '600px',
+    heightSize: '1100px',
+  },
+} as ComponentMeta<typeof Card>;
+
+const cards = [
```
Comment (caarlosdamian) [inline src/components/card/Card.stories.tsx:24]: move this to utils

File: src/components/card/card.css [hunk 5]
```diff
@@ -0,0 +1,144 @@
+
+.container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 32px 56px;
+  gap: 24px;
+  background-color: #ffffff;
```
Comment (caarlosdamian) [inline src/components/card/card.css:8]: border-radius???

File: src/components/card/card.css [hunk 6]
```diff
@@ -0,0 +1,144 @@
...
+  border-radius: 20px;
+  font-family: 'Nunito Sans', 'Helvetica Neue', Helvetica, Arial, sans-serif;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 40px;
+  gap: 40px;
+}
+
+.profile-image {
```
Comment (caarlosdamian) [inline src/components/card/card.css:22]: remove unnecesary styles that are not related to Card component move it to utils/utils.css 

please check figma desing card is not similar at all

File: src/components/card/card.css [hunk 7]
```diff
@@ -0,0 +1,144 @@
...
+  width: 128px;
+  height: 128px;
+}
+
+.name-card {
```
Comment (caarlosdamian) [inline src/components/card/card.css:27]: dont use flex this is only text text-aling:center;

File: src/components/card/card.css [hunk 8]
```diff
@@ -0,0 +1,144 @@
+
+.container {
+  display: flex;
```
Comment (caarlosdamian) [inline src/components/card/card.css:3]: remove display flex , the card is only a container base on widht and hight , user will add another container to styled as they want

File: src/components/card/card.css [hunk 9]
```diff
@@ -0,0 +1,144 @@
...
+  flex-direction: column;
+  align-items: center;
+  padding: 32px 56px;
```
Comment (caarlosdamian) [inline src/components/card/card.css:6]: add padding be 32px for all

File: src/components/card/card.css [hunk 10]
```diff
@@ -0,0 +1,144 @@
...
+  text-align: center;
+  color: #272727;
+  margin: 0;
+}
+
+.icon-container {
+  display: flex;
+  flex-direction: row;
+  align-items: flex-start;
+  padding: 0px;
```
Comment (caarlosdamian) [inline src/components/card/card.css:72]: remove px just padding: 0;

File: src/components/card/card.css [hunk 11]
```diff
@@ -0,0 +1,144 @@
...
+  gap: 24px;
+}
+
+.paragraph-container {
+  display: flex;
+  flex-direction: row;
```
Comment (caarlosdamian) [inline src/components/card/card.css:78]: dont need this since display-flex has flex-direction:row as default

File: src/components/card/card.css [hunk 12]
```diff
@@ -0,0 +1,144 @@
...
+.profile-image {
+  width: 128px;
+  height: 128px;
+}
+
+.name-card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 0px;
```
Comment (caarlosdamian) [inline src/components/card/card.css:31]: remove px

File: src/components/card/Card.stories.tsx [hunk 13]
```diff
@@ -0,0 +1,143 @@
...
+
+export const Children = Template.bind({});
+
+const smallCard = [
+  { color: 'white', width: '436px', height: '495px', image: image1 },
+];
+
+const TemplateCard: ComponentStory<typeof Card> = (args) => (
+  <Card {...args}>
+    {smallCard.map((cardProps) => (
```
Comment (caarlosdamian) [inline src/components/card/Card.stories.tsx:101]: why this is not in the Card component????

File: src/components/card/Card.tsx [hunk 14]
```diff
@@ -0,0 +1,28 @@
...
+}
+
+export const Card = ({
+  children,
+  widthSize,
+  heightSize,
+  className,
+}: CardProps) => {
+  return (
+    <div
```
Comment (caarlosdamian) [inline src/components/card/Card.tsx:18]: dont bother with all the card imgs or style , Card component is just a div with a box-shadow and custom size and recive a children delete unncesary code css/assest/ingnore the figma desing the content was just a example just creatre 3 stories small,medium and larga but with a children and thats it , let me know if you have any questions

**Review by caarlosdamian — CHANGES_REQUESTED** [review]
comments

File: src/components/card/Card.stories.tsx (hunk 1 above)
Comment (DiegoVillasenorVilla) [reply src/components/card/Card.stories.tsx:4]: Ok, for the moment, this assets aren`t necessary

**Review by caarlosdamian — APPROVED** [review]
lgtm
